Utils/trigger_test_upload_flow.sh

```diff
@@ -86,6 +87,9 @@ while [[ "$#" -gt 0 ]]; do
   -o|--override-all-packs) _override_all_packs=true
     shift;;
 
+    -dsn|--sdk-nightly) _demisto_sdk_nightly=true
```

Comment:
```suggestion
```

Reply:
@ilaner Why not give the option to set the SDK when triggering test upload?